src/activities/extract-meca.ts

```diff
@@ -47,21 +47,6 @@ type ManifestItem = {
   instance: ManifestItemInstance[],
 };
 
-const extractFileContents = async (zip: JSZip, item: MecaFile, toDir: string): Promise<LocalMecaFile> => {
-  const content = await zip.file(item.path)?.async('base64');
-  if (content === undefined) {
-    throw Error(`MECA archive corrupted, expected ${item.path} from manifest, but it failed`);
-  }
-  const outputPath = `${toDir}/${item.path}`;
-  const outputDir = dirname(outputPath);
-  mkdirSync(outputDir, { recursive: true });
-  writeFileSync(outputPath, content, 'base64');
-  return {
-    ...item,
-    localPath: outputPath,
-  };
-};
-
 export const extractMeca = async (version: VersionedReviewedPreprint): Promise<MecaFiles> => {
   const tmpDirectory = await mkdtemp(`${tmpdir()}/epp_content`);
```

Comment:
I know this was pre-existing, but this surely means that we have the potential to end up with files overwriting each other. 

What do you think to 
```
const tmpDirectory = await mkdtemp(`${tmpdir()}/epp_content/${version.id}`);
``` 
?

Reply:
Actually, that's nonsense.